### ISSUE TITLE
Wanna be file attribute is item attribute

### ISSUE THREAD

**Issue by mcarton** [Manishearth/rust-clippy#770]
I just had a WTF moment with tests because I wrote:
```rust
#[deny(panic_params)]

fn foo() {
    …; //~ERROR tralalalalah
}

fn bar() {
    …; //~ERROR tralalalalah
}
```
and the test suite reported a an expected message *and* an expected error not found in `bar`.

Took me a while to notice the `deny` attribute is missing a `!` and only applies to `foo`. We could warn when there is a new line between an outer attribute and the first item. Should be similar to what we have in `suspicious_else_formatting`.
